app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/views/PlaylistPreviewItem.kt

```diff
@@ -91,7 +93,8 @@ fun PlaylistPreviewItem(
                             modifier = Modifier
                                 .clip(thumbnailShape)
                                 .align(alignment)
-                                .size(thumbnailSize)
+                                .height(53.8.dp)
+                                .width(52.dp)
```

Comment:
Setting sizes in `dp` could be buggy. 52 dp is not going to look the same on all devices. 
Instead we need to calculate the proper size in code or use match_parent (match constraints) somehow.
Please see:
<img width="575" alt="image" src="https://user-images.githubusercontent.com/11182351/196123276-9d3fa25d-6aae-41a0-8e53-1a93579d0c25.png">

Reply:
there should be no space between images